www/src/templates/template-docs-markdown.js

```diff
@@ -136,6 +137,9 @@ function DocsTemplate({ data, location, pageContext: { next, prev } }) {
                     See the issue relating to this stub on GitHub
                   </a>
                 )}
+                {page.frontmatter.showSignupForm && (
```

Comment:
You should be able to just import and use the <EmailCaptureFrom> in [MDX](https://www.gatsbyjs.org/docs/mdx/writing-pages/).

In `index.md`:

```md
import EmailCaptureForm from 'www/src/components/EmailCaptureForm'

...content...

<EmailCaptureForm signupMessage="..."/>
```